mpifileutils.rpmlintrc

```diff
@@ -1,7 +1,8 @@
 # This line is mandatory to access the configuration functions
 from Config import *
 
-# https://github.com/hpc/mpifileutils/issues/567
-addFilter(r'mpifileutils-.+\.x86_64: E: missing-call-to-setgroups-before-setuid \/usr\/lib64\/.+/bin/dcp')
-
+# https://github.com/rpm-software-management/rpmlint/issues/73
+addFilter(r'mpifileutils-.+-devel\..+: W: only-non-binary-in-usr-lib')
```

Comment:
Ahhh.  This is not quite ready to be removed yet as the [fix](https://github.com/rpm-software-management/rpmlint/commit/1056f04337a1b0e6e37d2fc29bdb500bc35b8fd7) for rpm-software-management/rpmlint#73 landed in _rpmlint_ 2.5.0:
![image](https://github.com/daos-stack/mpifileutils-pkg/assets/559317/92768509-acd3-400a-b842-acaca0dbf634)
and we (rather, Fedora) seems to be still using 2.4.0:

```
[2024-01-10T17:15:03.597Z] rpmlint: 2.4.0
```

I've [requested an update to 2.5.0 in Fedora 39](https://bugzilla.redhat.com/show_bug.cgi?id=2257744).